crates/wastebin_ctl/src/main.rs

```diff
@@ -236,6 +245,30 @@ async fn main() -> Result<()> {
 
             println!("{table}");
         }
+        Commands::Delete {
+            database,
+            identifier,
+        } => {
+            let ids: Vec<_> = identifier
+                .iter()
+                .map(|ident| match Id::from_str(ident) {
+                    Ok(i) => i,
+                    Err(_) => {
+                        eprintln!("Invalid identifier `{ident}`");
+                        exit(1);
+                    }
+                })
```

Comment:
Same as in the other comment. Use `?` and anyhow's context.

Reply:
done